core/src/main/scala/org/broadinstitute/workbench/hamm/dao/WorkflowMetadataDAO.scala

```diff
@@ -3,11 +3,39 @@ package org.broadinstitute.workbench.hamm.dao
 import cats.effect.Sync
 import org.broadinstitute.workbench.hamm.model.CromwellMetadataJsonCodec._
 import org.broadinstitute.workbench.hamm.model.{MetadataResponse, WorkflowId}
+import org.http4s.Method._
 import org.http4s.circe.CirceEntityDecoder._
 import org.http4s.client.Client
+import org.http4s.client.dsl.Http4sClientDsl
+import org.http4s.headers._
+import org.http4s.{AuthScheme, Credentials, Uri}
 
-class WorkflowMetadataDAO[F[_]: Sync](httpClient: Client[F]) {
+// This is fallback for serving cost API when we don't find cost info in database
+class WorkflowMetadataDAO[F[_]: Sync](httpClient: Client[F]) extends Http4sClientDsl[F] {
   def getMetadata(workflowId: WorkflowId): F[MetadataResponse] = {
-    httpClient.expect[MetadataResponse](s"https://cromwell.dsde-alpha.broadinstitute.org/api/workflows/v1/${workflowId.uuid}/metadata") //TODO: figure out how to get the url properly
+    val uri = Uri.unsafeFromString(s"https://cromwell.dsde-alpha.broadinstitute.org/api/workflows/v1/${workflowId.uuid}/metadata")
+        .withQueryParam("includeKey", List(
+          "id",
+          "start",
+          "end",
+          "labels",
+          "executionEvents",
+          "runtimeAttributes",
+          "jobId",
+          "preemptible",
+          "callCaching:hit",
+          "hit",
+          "jes",
+          "papi2",
+          "executionStatus",
+          "backend")
+        )
+        .withQueryParam("expandSubWorkflows", true)
+
+    val request = GET(
+      uri,
+      Authorization(Credentials.Token(AuthScheme.Bearer, "ya29.Gl2xBueI7wZeIyZ7pOOnMqf-sVum_dF3mbdzzS4ZnSVlcEC6TDA17Z_fWqWnAJIY1GFD7zWTrfGdRZXe5TKru8EymyT9NFnJVNniGbimkC2-DejAI6zNTHxJlA17T3s")) //TODO: figure out proper authentication
+    )
+    httpClient.expect[MetadataResponse](request)
```

Comment:
I assume that this is just temporary until rawls starts pumping metadata and hamm can process it... I expect this to be removed at some point

Reply:
@dvoet  I don't know for sure. What we do if user asks for a workflow that we don't have its cost in database?

Reply:
404 error?

Reply:
architecturally, the only thing that should talk to cromwell directly is cromiam. So if we really wanted to do this we would have to talk to cromiam and we would have to use the user's token since cromiam will enforce security.

But really I think this opens up a performance problem... instead of processing metadata behind the scenes we will be doing it in real time with the user waiting. Plus we can potentially be processing metadata from old workflows with current price list.

Reply:
I understand this is not ideal. I'm fine with 404 and state we only support calculating cost for workflows run after hamm is live

Reply:
fyi. I was having a meeting with Ilyana, and brought this up with her. she's totally fine with only support calculating cost for workflows run after hamm is live.

I might delete this file later at some point, but it's still useful to have it for now.